<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetAttrDialog</class>
 <widget class="QWidget" name="PetAttrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/images/attrbk);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="1" colspan="3">
     <widget class="QLabel" name="label_dengji">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 20pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_26">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>科技熟练</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_gedang">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_tongshuai">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_shunfa">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QLabel" name="label_25">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/labbk);
color: rgb(137, 146, 166);
font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>技能属性</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_16">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>格挡</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_19">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>攻距</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>敏捷</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
color: rgb(99, 120, 124);
font: 20pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>等级</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/labbk);
color: rgb(137, 146, 166);
font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>基础属性</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_10">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>统帅</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_liliang">
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>力量</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="label_30">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>掘金</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_yidong">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_23">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>瞬发</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>智力</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_zhili">
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_minjie">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_keji">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_21">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>移动</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_juejin">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/labbk);
color: rgb(137, 146, 166);
font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>增幅属性</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_gongju">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_28">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>跳距</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_tiaoju">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_17">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>暴击</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_baoji">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_14">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>团战指挥</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_tuanzhan">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="label_31">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(99, 120, 124);</string>
      </property>
      <property name="text">
       <string>超能力</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QLabel" name="label_chaonengli">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;Arial&quot;;
color: rgb(187, 187, 187);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
